mixins/demo/localize-mixin-mission.js

```diff
@@ -1,22 +1,23 @@
+import '../../components/link/link.js';
 import { html, LitElement } from 'lit';
 import { LocalizeDynamicMixin } from '../../mixins/localize-dynamic-mixin.js';
 
 class Mission extends LocalizeDynamicMixin(LitElement) {
 
 	static get localizeConfig() {
 		const langResources = {
-			'en': { mission: 'Transforming the way [a]the world[/a] learns' },
-			'fr': { mission: 'Transformer la façon dont [a]le monde[/a] apprend' }
+			'en': { mission: 'Transforming the way <my-link>the <br></br> <b>world</b></my-link> learns' },
+			'fr': { mission: 'Transformer la façon dont <my-link>le <br></br> <b>monde</b></my-link> apprend' }
```

Comment:
One drawback is definitely that self-closing tags aren't supported - just using the opening `<br>` will result in an error (`UNMATCHED_CLOSING_TAG`), and using `<br/>` or `<br />` will just display that text.

Reply:
@dbatiste Makes a good point about using `<p></p>`, though, which one of my accessibility-minded teammates polices too. Inputs should be core components anyway. Images and anything else hopefully would be rare and tolerable.